changelogs/fragments/1446-backup_plan-add-new-module.yml

```diff
@@ -0,0 +1,3 @@
+trivial:
```

Comment:
changelog is not necessary when we add new modules. It gets added to the release summary during the release.

Reply:
The tests have been inconsistent in handling this :)  trivial can stay, just for the sake of not tripping over a test

Reply:
Yeah sanity tests failed without it

Reply:
Or um, maybe not sanity tests....I'm still learning what all the various checks are. But _something_ was failing :)